pkg/controller/clone-controller.go

```diff
@@ -288,11 +288,12 @@ func (r *CloneReconciler) updatePvcFromPod(sourcePod *corev1.Pod, pvc *corev1.Pe
 
 	log.V(3).Info("Pod phase for PVC", "PVC phase", pvc.Annotations[AnnPodPhase])
 
-	if podSucceededFromPVC(pvc) && pvc.Annotations[AnnCloneOf] != "true" {
+	if podSucceededFromPVC(pvc) && pvc.Annotations[AnnCloneOf] != "true" && sourcePodFinished(sourcePod) {
```

Comment:
so, previously the AnnCloneOf was added too early and because of this the next Reconcile of the clone PVC was skipped (line 185) 

now, AnnCloneOf is added at the very end,

Reply:
This would make a good info for a PR or commit message, besides that 
/lgtm

Reply:
I've updated the description.